docs/collector.time.md

```diff
@@ -0,0 +1,45 @@
+# time collector
+
+The time collector exposes the Windows Time Service metrics. Note that the Windows Time Service must be running, else metric collection will fail.
+If the Windows Time Service is stopped after collection has started, collector metric values will reset to 0.
+
+|||
+-|-
+Metric name prefix  | `time`
+Data source         | Perflib
+Enabled by default? | No
```

Comment:
This would make sense to me to enable by default, any reason we shouldn't?

Reply:
Collector is dependant on the Windows Time Service, which wasn't enabled by default on my Windows 10 VM.
Would that be a blocker to setting this as a default collector?

Reply:
Ah, okay. I had it enabled on the systems I checked quickly, but if it isn't a default then let's keep this one non-default as well.